packages/web/src/AMSTAR2/checklist.js

```
@@ -296,13 +296,13 @@ export function exportChecklistsToCSV(checklists) {
-        });
-      });
-    });
-  });
+        }
+      }
+    }
+  }
 
   // CSV encode
-  const escape = val => `"${String(val).replace(/"/g, '""').replace(/\n/g, ' ')}"`;
+  const escape = val => `"${String(val).replaceAll('"', '""').replaceAll('\n', ' ')}"`;
```

Biome (2.1.2)

[error] Do not shadow the global "escape" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)